### PR TITLE
feat: increase hitslop of section title

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [https://www.notion.so/artsy/Art-Advisor-Home-View-Feedback-abc1123548504ae58051405627fb6c9f] 

### Description

This PR updates the hitslop of the section title to make it easier to tap.

| Before | After |
|--------|--------|
| Simulator Screenshot - iPhone 16 Pro - 2024-09-26 at 18 01 43 | Simulator Screenshot - iPhone 16 Pro - 2024-09-26 at 18 01 35 |

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- increase hitslop of section titles - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (increase hitslop of section titles - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ca6e8bad8d9d241fa37594fe10671147cbf5fe9c
</p>

File: src/app/Components/SectionTitle.tsx [hunk 1]
```diff
@@ -5,7 +5,11 @@ import { TouchableOpacity } from "react-native"
 const Wrapper: React.FC<{ onPress?(): any }> = ({ onPress, children }) => {
   if (onPress) {
     return (
-      <TouchableOpacity onPress={onPress} testID="touchable-wrapper">
+      <TouchableOpacity
+        onPress={onPress}
+        testID="touchable-wrapper"
+        hitSlop={{ top: 10, bottom: 10 }}
+      >
```
Comment (anandaroop) [inline src/app/Components/SectionTitle.tsx:12]: **question (non-blocking):** Just wanted to confirm what happens when the hit areas overlap:



In this case, does the _1 new work_ activity item "win"?

**Review by anandaroop — APPROVED** [review]
Awesome, thanks!

File: src/app/Components/SectionTitle.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/app/Components/SectionTitle.tsx:12]: that's a good question - usually, elements that come below other elements have a higher zIndex so they will be the ones that capture the touches.


https://github.com/user-attachments/assets/04cc6244-f7f0-41da-a985-7d090d784161

**Pull request merged**